src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -569,26 +577,26 @@ public ClusterDaemon(ClusterSettings settings)
 
             Context.ActorOf(Props.Create<ClusterHeartbeatReceiver>().WithDispatcher(settings.UseDispatcher), "heartbeatReceiver");
 
+            Receive<InternalClusterAction.GetClusterCoreRef>(msg => _coreSupervisor.Forward(msg));
+            Receive<InternalClusterAction.AddOnMemberUpListener>(msg =>
+                       Context.ActorOf(Props.Create(() => new OnMemberStatusChangedListener(msg.Callback, MemberStatus.Up)).WithDispatcher(_settings.UseDispatcher).WithDeploy(Deploy.Local)));
+            Receive <InternalClusterAction.AddOnMemberRemovedListener>(msg =>
```

Comment:
Handler for a new message type (`AddOnMemberRemovedListener`) - used to help perform cleanup events when the current node is removed from the `Cluster`.